src/charts.js

```diff
@@ -305,22 +319,44 @@ function dateToHumanString(date) {
 }
 
 function downloadsURL(pkg, from, to) {
-  return '/downloads/range/' + dateToString(from) + ':' + dateToString(to) + '/'
+  return 'http://npm-stat.com/downloads/range/' + dateToString(from) + ':' + dateToString(to) + '/'
```

Comment:
Because I'm testing it on a local server, don't mind it :). Once it's ready to merge I'll remove unnecessary commits and prettify current ones.